policy/handler.go

```diff
@@ -218,3 +291,46 @@ func (h *Handler) updateHandler(current, next *Policy) {
 		h.ticker = time.NewTicker(next.EvaluationInterval)
```

Comment:
btw, this discards the current evaluation interval by resetting the ticker to zero. that's fine (and probably moot) for "short" evaluation intervals. for "longer" ones, maybe that would be undesirable? in which case, we end up with a conundrum where we need to have a short (potentially, zero) first interval and then transition to the new interval (probably would want to use a custom Ticker implementation).

Reply:
I think the key here is the order of magnitude of "short" and "long". Horizontal autoscalers are usually targeted for long (multi-day) running services. This condition would only be triggered by a change in `evaluation_interval`, which would probably be set in the order of minutes at most, which sounds long when doing something like a quick test, but it's actually short when compared to the lifecycle of the service.